Products/Zuul/catalog/indexable.py

```diff
@@ -375,7 +376,10 @@ def idx_hwManufacturer(self):
     def idx_serialNumber(self):
         return self.hw.serialNumber
 
-
+    @indexed(StringFieldType(indexed=True, stored=True), attr_query_name="snmpLastCollection")
```

Comment:
Any reason to not index the timestamp as a `FloatFieldType`? Timestamps are generally handled as numerical values, not strings.

Reply:
As we discussed, added as a DoubleFieldType to avoid potential precision loss.